app/domain/people_skills_filter.rb

```diff
@@ -1,16 +1,17 @@
 # frozen_string_literal: true
 
 class PeopleSkillsFilter
-  attr_reader :entries, :rated, :level
+  attr_reader :entries, :rated, :level, :skill_id
 
-  def initialize(entries, rated, level = nil)
+  def initialize(entries, rated, level = nil, skill_id = nil)
```

Comment:
I just realized, assigning nil as default value does absolutely nothing. If you pass nothing, the value is nil. You can just remove the default value assignment

Reply:
If these are not assigned something by default, I get an argument error when calling it without a level and skill_id.

Reply:
I see, didn't know that. In that case it's fine :)